events/0316_git.html

```diff
@@ -0,0 +1,215 @@
+                    <aside class="notes"></aside>
+                </div>
+            </section>
+
+            <section>
+                <h3>Community Events</h3>
+                <ul class="box copy--xsmall left" style="width: 48%">
+                    <li><strong>SEMjs - Native Mobile Apps with React Native</span></strong><br /><small>Monday 3/14 @ Pillar</small></li>
+                    <li><strong><span>Mom:Code - Intro to Coding</span></strong><br /><small>Tuesday, 3/15 @ Grand Circus - Detroit</small></li>
+                    <li><strong><span>Connect over Coffee, Learn a NEW Networking Strategy</span></strong><br /><small>Tuesday, 3/15 @ Grand Traverse Pie Ccompany</small></li>
+                    <li><strong>Ann Arbor New Tech Meetup</strong><br /><small>Tuesday, 3/15 @ UM Law School, South Hall</small></li>
+                    <li><strong><span>Block_Element--Magic: CSS Modularity for the Masses</span></strong><br /><small>Wednesday, 3/16 @ Grand Circus - Detroit</small></li>
+                    <li><strong>Free Code Camp - Coffee and Code</strong><br /><small>Wednesday, 3/16 @ Espresso Royal - State St.</small></li>
+                    <li><strong><span>User Experience Hand-On Workshop</span></strong><br /><small>Wed. 3/16 &amp; Thurs. 3/17 @ Grand Circus - Detroit</small></li>
+                </ul>
+                <ul class="box copy--xsmall right" style="width: 48%">
+                    <li><strong><span>PHP Lightning Talks</span></strong><br /><small>Thursday 3/17 @ Nutshell</small></li>
+                    <li><strong><span>Java Bootcamp Demo Day</span></strong><br /><small>Friday, 3/18 @ Grand Circus - Detroit</small></li>
+                    <li><strong><span>Coding Bootcamp Open House</span></strong><br /><small>Friday, 3/18 @ Grand Circus - Detroit</small></li>
+                    <li><strong><span>Ladies That UX Happy Hour</span></strong><br /><small>Tuesday 3/22 @ Bobcat Bonnies, Detroit</small></li>
+                    <li><strong>VoiceOps; Command &amp; Control your AWS Environements</strong><br /><small>Thursday, 3/17 @ Duo Security</small></li>
+                    <li><strong><span>Web Analytics Wednesday</span></strong><br /><small>Wednesday, 3/23 @ AADL Downtown</small></li>
+                    <li><strong><span>Coffee House Coders</span></strong><br /><small>Wednesday, 3/23 @ Espresso Royal - State St.</small></li>
+                    <li><strong><span class="blue">self.conference</span></strong><br /><small>Friday, 5/20 and Saturday, 5/21 @ Motor City Casino<br/><span class="red">Use code "gdi-discount" for 15% off ticket price!</span></small></li>
+                    <li class="green"><em>Plus soooo many more...</em></li>
+                </ul>
+                <p class="clear copy--small"><a href="http://bit.ly/gdiaa-community">bit.ly/gdiaa-community</a></p>
+                <aside class="notes"></aside>
+            </section>
```

Comment:
Can you make sure the group name is present before the event title - I think some of these are missing it. But this just makes it easier for people to find these events after they leave.